db/migrate/20220609103801_add_language_to_users.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddLanguageToUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users do |t|
+      t.string :language, default: ''
```

Comment:
I think this should be `null: false`

Reply:
Since, this field is relevant only to the client app shouldn't it be saved without any validation from the backend?

>NOTE: This will fail 82 examples temporarily until subsequent PRs gets merged.

Reply:
This helps spot all the actions that intervenes in the user creation, Is that why we should fail any lack of language setting upon creation?

Reply:
I see what you mean,
Thank you Ahmad!